sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResourceApi.java

```diff
@@ -0,0 +1,340 @@
+                      + "| $eq | Equal To | \n "
+                      + "| $ne | Not Equal To | \n "
+                      + "| $in | Contained In | \n "
+                      + "| $contains | Contains the given element (for lists or sets) or value (for maps) | \n "
+                      + "| $containsKey | Contains the given key (for maps) | \n "
+                      + "| $containsEntry | Contains the given key/value entry (for maps) | \n "
+                      + "| $exists | Returns the rows whose column (boolean type) value is true | ",
+              required = true)
+          @QueryParam("where")
+          final String where,
+      @ApiParam(value = "URL escaped, comma delimited list of keys to include")
+          @QueryParam("fields")
+          final String fields,
+      @ApiParam(value = "Restrict the number of returned items") @QueryParam("page-size")
+          final int pageSizeParam,
+      @ApiParam(value = "Move the cursor to a particular result") @QueryParam("page-state")
+          final String pageStateParam,
+      @ApiParam(value = "Unwrap results", defaultValue = "false") @QueryParam("raw")
+          final boolean raw,
+      @ApiParam(value = "Keys to sort by") @QueryParam("sort") final String sort,
+      @Context HttpServletRequest request);
+
+  @Timed
+  @GET
+  @ApiOperation(
+      value = "Get row(s)",
+      notes = "Get rows from a table based on the primary key.",
+      response = Sgv2RowsResponse.class)
+  @ApiResponses(
+      value = {
+        @ApiResponse(code = 200, message = "OK", response = Sgv2RowsResponse.class),
+        @ApiResponse(code = 400, message = "Bad Request", response = RestServiceError.class),
+        @ApiResponse(code = 401, message = "Unauthorized", response = RestServiceError.class),
+        @ApiResponse(
+            code = 500,
+            message = "Internal server error",
+            response = RestServiceError.class)
+      })
+  @Path("/{primaryKey: .*}")
+  public Response getRows(
+      @ApiParam(
+              value =
+                  "The token returned from the authorization endpoint. Use this token in each request.",
+              required = true)
+          @HeaderParam("X-Cassandra-Token")
+          String token,
+      @ApiParam(value = "Name of the keyspace to use for the request.", required = true)
+          @PathParam("keyspaceName")
+          final String keyspaceName,
+      @ApiParam(value = "Name of the table to use for the request.", required = true)
+          @PathParam("tableName")
+          final String tableName,
+      @ApiParam(
+              value =
+                  "Value from the primary key column for the table. Define composite keys by separating values with slashes (`val1/val2...`) in the order they were defined. </br> For example, if the composite key was defined as `PRIMARY KEY(race_year, race_name)` then the primary key in the path would be `race_year/race_name` ",
```

Comment:
Can we split this with line breaks?

Reply:
I can try to see how Google indenter handles that.